docs/use/start.md

```diff
@@ -112,4 +112,7 @@ Then for parsing and output rendering:
 * - `nb_output_stderr`
   - `show`
   - One of 'show', 'remove', 'warn', 'error' or 'severe', [see here](use/format/stderr) for details.
+* - `nb_merge_streams`
+  - `False`
+  - If `True`, ensure all stdout / stderr output streams are merged into single outputs. This ensures deterministic outputs.
```

Comment:
This explanation is perhaps worth to clarify further. Which aspect of the outputs becomes deterministic?

Reply:
Too slow lol, new version already out (wanted it for https://sqla-tutorials-nb.readthedocs.io).
Feel free to make a PR with the change though, and I'll merge

Reply:
I agree, perhaps 5 minutes left to review of a new config option is too short. Maybe something to consider for the future.

Also I can't make this PR exactly because I don't understand what result does merging of the streams achieve. Given that I'm somewhat familiar with nbformat and the involved stack, this is possibly true for many others.

Reply:
The result is "documented" by the test and it output (see PR files changed).
It is a common pattern, already used in places like: https://github.com/computationalmodelling/nbval/blob/master/nbval/plugin.py, and was already planned in https://github.com/executablebooks/jupyter-book/issues/973

Reply:
Thanks, I understand the purpose of the feature now. I'll trust your judgement call that looking at the source and the test is good enough as far as documentation is concerned.

Reply:
Yeh, I'm certainly open to improved documentation, and we will probably iterate on that more in https://github.com/executablebooks/jupyter-book/pull/1448#issuecomment-933049230

The results, e.g. here https://sqla-tutorials-nb.readthedocs.io/en/latest/metadata.html#emitting-ddl-to-the-database, speak for themselves though: before all the output lines were in separate `div` boxes 😄

Reply:
> The result is "documented" by the test and it output (see PR files changed).
> It is a common pattern, already used in places like: https://github.com/computationalmodelling/nbval/blob/master/nbval/plugin.py, and was already planned in [executablebooks/jupyter-book#973](https://github.com/executablebooks/jupyter-book/issues/973)

Thanks @chrisjsewell -- this is a great improvement 👍 

Thanks @akhmerov for your comments -- I am also not sure what `This ensures deterministic outputs.` means here. 

Just my `2cents`--  I find `documentation` in tests a sub-optimal form of documentation as it requires reading tests, which typically means you need to understand the `test` infrastructure with is non-trivial in many cases.

Reply:
Its deterministic because, instead of having a random (non-deterministic) number of stdout/stderr outputs, you have only a maximum of one for each. As explained in https://github.com/executablebooks/jupyter-book/issues/973

Reply:
I agree that this could be documented more clearly for end-users - if we think that this is a useful feature, and if it is turned off by default, then we should have a section that is easily discoverable for a user that would benefit from this feature.

For example, we could create a parent category of ``## `stdout`/`stderr` `` [around here](https://myst-nb.readthedocs.io/en/latest/use/formatting_outputs.html#removing-stdout-and-stderr), then create a nested section like:

````
### Group `stdout` and `stderr` outputs into a single stream

Notebooks may print multiple things to `stdout` and `stderr`. For example, if a cell prints status updates throughout its execution, each of these is often printed to `stdout`. By default, these outputs may be split across multiple items, and will be rendered as separate "chunks" in your built documentation.

If you'd like each of the outputs in `stderr` and `stdout` to be merged into a single stream for each, use the following configuration:

```
nb_merge_streams = True
```

This will ensure that all `stderr` and `stdout` outputs are merged into a single group.
This also makes the cell outputs more deterministic because 
````

Reply:
I'm just gonna open a PR to propose these changes since this one is already merged.